common-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM agpipeline/base-image:1.0
+FROM agpipeline/base-image:1.1
```

Comment:
I would suggest using `latest` here so that changes are caught earlier. Another approach could be to use environment variables in conjunction with TravisCI build variables to default to `latest` unless a specific version is specified?

Reply:
Ok this is fixed, it is now set to latest

Reply:
This is in the common-image dockerfile. I haven't changed anything here myself, I'm not even sure where that `1.0` vs `1.1` happened. Should I still change to `latest` anyway along with the test-image?

Reply:
My bad. I missed where this file was